source/controller/src/map_ctrl_main.c

```diff
@@ -64,9 +69,9 @@ static bool g_signal_stop = false;
 ########################################################################*/
 static void signal_stop_handler(UNUSED int signum)
 {
-    if (signum != SIGINT) {
-        ssp_stack_backtrace();
-    }
+    // if (signum != SIGINT) {
+    //     ssp_stack_backtrace();
+    // }
     g_signal_stop = true;
```

Comment:
Please add Breakpad support, after rdk-easymesh-controller.bb recipe file is merged.